contracts/Comptroller/Comptroller.sol

```diff
@@ -1405,7 +1405,7 @@ contract Comptroller is ComptrollerV11Storage, ComptrollerInterfaceG2, Comptroll
 
         xvs.approve(getXVSVTokenAddress(), amount);
         require(
-            VBep20Interface(getXVSVTokenAddress()).mintBehalf(user, amount) == uint(Error.NO_ERROR),
+            VBep20InterfaceV2(getXVSVTokenAddress()).mintBehalf(user, amount) == uint(Error.NO_ERROR),
```

Comment:
The changes in comptroller and VAI should not be there as the function signatures of all the functions in vToken is same so we can use the same interface `VBep20Interface` to call to vToken.